### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by Alusp**
# I implemented the following in this PR:

- [ ] Create a new React app called 'hello-react-front-end'
- [ ] Add Gitflow
- [ ] Add Linters configurations
- [ ] Initialize my project with Git.
- [ ] Create the App component with react-router.
- [ ] Create the Greeting component that will display a greeting. 
- [ ] Set it up as a route in my App component.
- [ ] Create a store, an action and a reducer that will connect to my API endpont to get the random greeting.
- [ ] Display the random greeting in my Greeting component.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import Greeting from './Greeting';
+import store from './redux/config/store';
+
+const App = () => (
+  <Provider store={store}>
+    <Greeting />
+  </Provider>
+);
+
+export default App;
```
Comment (sumairq) [inline src/App.js]: - [ ] Kindly add react-router to your React application and ensure that the `App.js` component has a route that leads to the Greeting component. Cheers 👍

**Review by sumairq — CHANGES_REQUESTED** [review]
### Hi @Alusp   , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Professional README
✔️Correct Git flow
✔️Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Alusp) [reply src/App.js]: Alright, thanks very much for the review @sumairq , I just resolved it

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Alusp 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
✔️Linters are passing✔️
✔️Professional README✔️
✔️Correct Git flow✔️
✔️Descriptive PR✔️

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**